src/androidTest/java/com/mixpanel/android/viewcrawler/EditProtocolTest.java

```diff
@@ -25,8 +25,8 @@
 
 
 public class EditProtocolTest extends AndroidTestCase {
-    private static class MockOnErrorListener implements ViewVisitor.OnErrorListener {
-        public MockOnErrorListener() {
+    private static class MockOnLayoutErrorListener implements ViewVisitor.OnLayoutErrorListener {
```

Comment:
We should probably change the name of the `onError` method to `onLayoutError` in this case.